src/styles/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin animation($duration, $timing-function) {
+  animation-name: move;
```

Comment:
The `animation-name` property is set to `move`, but the keyframes for this animation are not defined in this file. Ensure that the `move` animation is defined in your styles to avoid issues.